pycbc/events/ranking.py

```diff
@@ -4,6 +4,49 @@
 import numpy
 
 
+def newsnr_sgveto_psdvar_scaled(snr, bchisq, sgchisq, psd_var_val):
+    """ Combined SNR derived from NewSNR, Sine-Gaussian Chisq and PSD
+    variation statistic """
+    nsnr = numpy.array(newsnr_sgveto(snr, bchisq, sgchisq), ndmin=1)
+    psd_var_val = numpy.array(psd_var_val, ndmin=1)
+    # 1.2 is the expected maximum psd_var_val over gaussian noise.
+    nsnr = nsnr / (psd_var_val)**0.33
+
+    # If snr input is float, return a float. Otherwise return numpy array.
+    if hasattr(snr, '__len__'):
+        return nsnr
+    else:
+        return nsnr[0]
+
+def newsnr_sgveto_psdvar_scaled_threshold(snr, bchisq, sgchisq, psd_var_val):
+    """ Combined SNR derived from NewSNR and Sine-Gaussian Chisq"""
+    nsnr = newsnr_sgveto_psdvar_scaled(snr, bchisq, sgchisq, psd_var_val)
+    nsnr = numpy.array(nsnr, ndmin=1)
+    nsnr[bchisq > 2.0] = 1
+
+    # If snr input is float, return a float. Otherwise return numpy array.
+    if hasattr(snr, '__len__'):
+        return nsnr
+    else:
+        return nsnr[0]
+
+def newsnr_sgveto_psdvar_com(snr, bchisq, sgchisq, psd_var_val):
+    """ Combined SNR derived from NewSNR, Sine-Gaussian Chisq and PSD
+    variation statistic """
+    nsnr = numpy.array(newsnr_sgveto(snr, bchisq, sgchisq), ndmin=1)
+    psd_var_val = numpy.array(psd_var_val, ndmin=1)
+
+    # 1.2 is the expected maximum psd_var_val over gaussian noise.
+    lgc = psd_var_val >= 1.2
+    nsnr2 = nsnr / (psd_var_val)**0.33
+    nsnr2[lgc] = nsnr[lgc] / psd_var_val[lgc]
```

Comment:
Hardcoded stuff here too.

Reply:
Are we going to use this statistic for anything? BTW, I thought the Portsmouth psd stat penalised everything above 1.2 by the square root of psd var.. I'd support having that coded up.

Also what is _com supposed to mean?

Reply:
I'm removing this. It was a random experiment. 1.2 is already coded and on masater as newsnr_sgveto_psdvar (further down file).